src/color/mod.rs

```diff
@@ -387,7 +395,7 @@ impl Color {
                 color
             });
 
-            Id::from_ptr(color)
```

Comment:
> Hmmm, so the examples crashing really just appears to be something in `color/appkit_dynamic_color.rs`. For whatever reason, when going to set it as the layer background color in `view/appkit.rs`, it changes out underneath from `CacaoDynamicColor` to a private `_NSTrackingAreaAKViewHelper` instance.
>
> That symbol is, unfortunately, not simple to search. Offhand I'm unsure if there's a particular change in `objc2` that'd cause this - any ideas? Maybe something funky with the way the pointer for the color is vended in `color/mod.rs:to_objc`?

This is what changed, previously the color was double-retained and leaked.

You can get the same behaviour on `trunk` with:
```diff
diff --git a/src/color/mod.rs b/src/color/mod.rs
index 0f46cd2..06abad8 100644
--- a/src/color/mod.rs
+++ b/src/color/mod.rs
@@ -387,7 +387,7 @@ impl Color {
                 color
             });
 
-            Id::from_ptr(color)
+            Id::from_retained_ptr(color)
         })))
     }
 
```

The issue is that `NSView::set_background_color` (or `Listview::set_background_color`, or really anything that uses `set_ivar`) doesn't retain the color before setting it as the instance variable, so once the `Color` drops, and with it sets the reference count for the background color to 0 (so the stuff you were seeing `_NSTrackingAreaAKViewHelper` was just because the pointer was reused for something else).